deeprank_gnn/models/query.py

```diff
@@ -453,76 +649,111 @@ def _set_amino_acid(graph: Graph, variant_residue: Residue,
+                node.features[FEATURENAME_POLARITYDIFFERENCE] = numpy.zeros(
+                    len(atom.residue.amino_acid.polarity.onehot)
+                )
 
     @staticmethod
-    def _set_pssm(graph: Graph, variant_residue: Residue,
-                  wildtype_amino_acid: AminoAcid, variant_amino_acid: AminoAcid):
+    def _set_pssm(
+        graph: Graph,
+        variant_residue: Residue,
+        wildtype_amino_acid: AminoAcid,
+        variant_amino_acid: AminoAcid,
+    ):
 
         for node in graph.nodes:
             atom = node.id
 
             pssm_row = atom.residue.get_pssm()
-            pssm_value = [pssm_row.conservations[amino_acid]
-                          for amino_acid in SingleResidueVariantAtomicQuery.amino_acid_order]
+            pssm_value = [
+                pssm_row.conservations[amino_acid]
+                for amino_acid in SingleResidueVariantAtomicQuery.amino_acid_order
+            ]
 
             if atom.residue == variant_residue:
 
-                node.features[FEATURENAME_PSSMDIFFERENCE] = pssm_row.get_conservation(variant_amino_acid) - \
-                                                            pssm_row.get_conservation(wildtype_amino_acid)
+                node.features[FEATURENAME_PSSMDIFFERENCE] = pssm_row.get_conservation(
+                    variant_amino_acid
+                ) - pssm_row.get_conservation(wildtype_amino_acid)
 
-                node.features[FEATURENAME_PSSMWILDTYPE] = pssm_row.get_conservation(wildtype_amino_acid)
-                node.features[FEATURENAME_PSSMVARIANT] = pssm_row.get_conservation(variant_amino_acid)
+                node.features[FEATURENAME_PSSMWILDTYPE] = pssm_row.get_conservation(
+                    wildtype_amino_acid
+                )
+                node.features[FEATURENAME_PSSMVARIANT] = pssm_row.get_conservation(
+                    variant_amino_acid
+                )
             else:
                 node.features[FEATURENAME_PSSMDIFFERENCE] = 0.0
-                node.features[FEATURENAME_PSSMWILDTYPE] = pssm_row.get_conservation(atom.residue.amino_acid)
-                node.features[FEATURENAME_PSSMVARIANT] = pssm_row.get_conservation(atom.residue.amino_acid)
+                node.features[FEATURENAME_PSSMWILDTYPE] = pssm_row.get_conservation(
+                    atom.residue.amino_acid
+                )
+                node.features[FEATURENAME_PSSMVARIANT] = pssm_row.get_conservation(
+                    atom.residue.amino_acid
+                )
 
             node.features[FEATURENAME_INFORMATIONCONTENT] = pssm_row.information_content
             node.features[FEATURENAME_PSSM] = pssm_value
 
     @staticmethod
     def _set_sasa(graph: Graph, pdb_path: str):
 
-        structure = freesasa.Structure(pdb_path)
-        result = freesasa.calc(structure)
+        structure = freesasa.Structure(pdb_path) # pylint: disable=c-extension-no-member
```

Comment:
Pylint imports modules to effectively identify valid methods and attributes. It was decided that importing c extensions that are not part of the python stdlib is a security risk and could introduce malicious code. So pylint gives me errors when we use freesasa (that is a C-library). Can this be a concern at some point?

Reply:
Deeprank runs on a dedicated server, where it will do calculations. It will not need root rights to do this.
SASA computations are necessary for deeprank to function. If you find an alternative, it would be acceptable to drop SASA.

Reply:
Ok so for now it's perfectly fine